front/app/containers/UsersEditPage/index.tsx

```diff
@@ -47,12 +48,18 @@ export default () => {
   const { data: appConfig } = useAppConfiguration();
   const authUser = useAuthUser();
   const loaded = appConfig !== undefined && authUser !== undefined;
+  const showEditPage =
+    loaded && !isNilOrError(appConfig) && !isNilOrError(authUser);
 
   if (loaded && !authUser) {
     clHistory.push('/');
   }
 
-  if (loaded && !isNilOrError(appConfig) && !isNilOrError(authUser)) {
+  if (showEditPage && authUser.attributes.confirmation_required) {
+    return <Unauthorized />;
+  }
```

Comment:
Just want to check that this is the proper way to handle unauthorized page visits?

Reply:
Yeah I would also do it like this!